README.md

```diff
@@ -76,52 +76,66 @@ The following pre-built docker images are available. For running dockers, check
 
 | Version | Cloud TPU VMs Docker |
 | --- | ----------- |
+2.0 | `gcr.io/tpu-pytorch/xla:r2.0_3.8_tpuvm` |
 1.13 | `gcr.io/tpu-pytorch/xla:r1.13_3.8_tpuvm` |
-1.12 | `gcr.io/tpu-pytorch/xla:r1.12_3.8_tpuvm` |
 nightly | `gcr.io/tpu-pytorch/xla:nightly_3.8_tpuvm` |
 nightly at date | `gcr.io/tpu-pytorch/xla:nightly_3.8_YYYYMMDD` |
 
 <br/>
 
-| Version | GPU CUDA 11.2 + Python 3.7 Docker |
+| Version | GPU CUDA 11.8 + Python 3.8 Docker |
```

Comment:
do we want to also mentioned 11.7 as a supported version here? any thoughts?

Reply:
There's a separate section starting on line 94 for CUDA 11.7, but I'm open to organizing these tables however you think would be clearest